app/pages/home-common/featured-project.jsx

```diff
@@ -16,11 +16,13 @@ const FeaturedProject = () => {
     <section className="home-featured">
       <h3 className="secondary-kicker">Featured Project</h3>
       <div className="home-featured-images">
-        <img role="presentation" src="./assets/featured-projects/featured-project-20170407-etchacell.jpg" />
+        <div><img role="presentation" src="./assets/featured-projects/featured-project-20170508-michiganzoomin01.jpg" /></div>
```

Comment:
Flexbox wasn't cooperating with plain `img` tags, but wrapping them in a `div` made them responsive.